scripts/us_cdc/environmental_health_toxicology/precipitation/scripts/parse_precipitation_index_test.py

```diff
@@ -32,25 +33,26 @@ class TestParsePrecipitationData(unittest.TestCase):
     Tests the functions in parse_precipitation_index.py.
     """
 
-    def test_clean_precipitation_data(self):
+    def test_clean_precipitation_data(self, test_csv, expected_csv):
         """
         Tests the clean_precipitation_data function.
         """
         print(module_dir_)
-        test_csv = os.path.join(module_dir_, 'test_data/small_Palmer.csv')
-        output_csv = os.path.join(module_dir_,
-                                  'test_data/small_Palmer_output.csv')
+        output_csv = test_csv[:-4] + '_output.csv'
         clean_precipitation_data(test_csv, output_csv)
-
-        expected_csv = os.path.join(module_dir_,
-                                    'test_data/small_Palmer_expected.csv')
         with open(output_csv, 'r') as test:
             test_str: str = test.read()
             with open(expected_csv, 'r') as expected:
                 expected_str: str = expected.read()
                 self.assertEqual(test_str, expected_str)
         os.remove(output_csv)
 
+def main():
```

Comment:
Unit tests should probably continue to use unittest.main().

Its useful to list the test files within the test file, so that, a command like `python3 -m unittest discover -v -s scripts -p *_test.py` (run from pre-submit tests) will exercise all these tests.

For example:

```
def test_clean_precipitation_data(self):
  TEST_CASE_FILES = [
    # Pairs of input CSV and expected CSV files.
    ('Palmer_Drought_Severity_Index_test_file.csv', 'Palmer_Drought_Severity_Index_test_file_expected_file.csv'),
    (..., ...),
    ...
  ]

  for test_csv, expected_csv in TEST_CASE_FILES:
    # Include the stuff now in test_clean_precipitation_data
    ...
```

Reply:
OK! switched back to using unittest.main() and listed the test case files in the script